test/integration/regression.js

```diff
@@ -1,4 +1,4 @@
-var assert = require('assert');
+﻿var assert = require('assert');
 var fs     = require('fs');
```

Comment:
Did this... pick up a UTF-8 BOM or something? I will see if I can fix that when I get another free moment.